### PR TITLE
FEAT(fov): random 90 deg rotations

### LINKED ISSUE #14: Implement Random90Rot?
It’d be good to have random rotations by +/-90 degrees about specified axes, analogous to the random flip implementation about the same.

### PR DESCRIPTION
**Pull request by balbasty**
Add random 90 degree rotations without resampling. Fixes #14 

@seannz Do you mind checking this branch out and making sure it works as you expect?

### CODE REVIEW

**Comment by seannz** [comment]
Yes sir, beta tester at your service.

**Comment by balbasty** [comment]
Hi @seannz any bug to report? Can I merge?

File: cornucopia/geometric.py [hunk 1]
```diff
@@ -1230,16 +1229,16 @@ def apply(self, x):
...
-                # not a label map -> apply PSF
-                kernel = torch.full(
-                    [nb_repeat, 1],
-                    1 / math.sqrt(nb_repeat),
-                    dtype=y.dtype, device=y.device
-                )
-                y = y.movedim(zindex, -1).unfold(-1, nb_repeat, nb_repeat)
-                y = y.matmul(kernel).matmul(kernel.t())  # PSF + replicate
-                y = y.flatten(-2).movedim(-1, zindex)    # [C, *oshape]
+            # if y.is_floating_point():
```
Comment (balbasty) [inline cornucopia/geometric.py:1232]: Why did you remove this test. Do you sometimes apply the PSF to label maps? I am surprised the code within the if block even runs when `y` is not float (the matmul should fail)

File: cornucopia/geometric.py [hunk 2]
```diff
@@ -1445,7 +1444,7 @@ def node2slice(x):
             x = torch.as_tensor(x, dtype=torch.float32).T  # [D, N]
             x = interpol.resize(x, shape=[nb_slices],
                                 interpolation=3, bound='replicate',
-                                prefilter=True).T  # [S, D]
+                                prefilter=False).T  # [S, D]
```
Comment (balbasty) [inline cornucopia/geometric.py:1447]: Why did you remove the interpolation?
